content/spin/kv-store-tutorial.md

```diff
@@ -433,12 +417,12 @@ Now let's build and deploy our Spin Application locally. Run the following comma
 $ spin build
 ```
 
-Now run the subsequent command to deploy your application: 
+Now, we have the option to add a key value pair to our store at application deployment time using the command `spin up --key-value`. This feature is helpful when you need to pass in data that's used in the initialization process. If you aren't interested in pre-seeding your key value store, then please use the `spin up` feature as usual. 
 
 <!-- @selectiveCpy -->
 
 ```bash
-$ spin up
+$ spin up --key-value connectionstring,34e5d0-42sd54-45c4a5-4a5s32
```

Comment:
Are you sure this flag exists?  I can't see it in the help text or the code, and get an error when I try to use it.

Reply:
Looks like it exists on deploy but not up?  (and the syntax there is `foo=bar` rather than `foo,bar`)